addons/op-badge/userscript.js

```diff
@@ -0,0 +1,17 @@
+export default async function({ addon, console }) {
+    const posts = document.querySelectorAll(".blockpost");
+
+    const op = await fetch(`https://scratchdb.lefty.one/v3/forum/topic/posts/${window.location.href.split("/")[5]}/0?o=oldest`).then(res => res.json());
+
+    for (const i of posts) {
```

Comment:
Could [addon.tab.waitForElement](https://scratchaddons.com/docs/reference/addon-api/addon.tab/#using-addontabwaitforelement) be used instead?

Reply:
@Samq64 waitForElements is not needed in forum posts (unless you set runAtComplete to false)

Reply:
Setting this as resolved as waitForElement is discouraged in this context.